manifests/06-sa.yaml

```diff
@@ -3,3 +3,9 @@ kind: ServiceAccount
 metadata:
   name: console-operator
   namespace: openshift-console-operator
+---
+apiVersion: v1
```

Comment:
Oh this is interesting.  Does the console not yet have a service account?

Reply:
Guess so:

```bash
oc get serviceaccount -n openshift-console
NAME       SECRETS   AGE
builder    2         28h
default    2         28h
deployer   2         28h

oc get serviceaccount -n openshift-console-operator
NAME               SECRETS   AGE
builder            2         28h
console-operator   2         28h
default            2         28h
deployer           2         28h
```

Reply:
yeah, only the default ones. For that reason I've created a new one